scripts/install/backend.sh

```diff
@@ -8,7 +8,7 @@ echo
 target=$1
 
 if [[ $target == *"prod"* ]]; then
-    pip install .
+    pip install -e .
```

Comment:
Please undo this change. We should not be installing for production with the `-e` flag.

Reply:
@ccarterlandis : I will see if taking that change out results in the `augur.config.json` not getting written out with `make install` ... I was able to replicate the problem on the current state of `master` using python 3.8.5 on mac OSX catalina.

Reply:
Then this is a red herring, if this "fixes" that issue then there is something else going horribly wrong with the installation that is the real issue. Adding the flag to install the package in editable mode is not and should not be required for production, only for development.